recipes-extended/xen/files/libxl-disable-dom0-qemu.patch

```diff
@@ -0,0 +1,27 @@
+libxl: do not start qemu in dom0 just for extra consoles
```

Comment:
Is this completely blocking qemu from running in dom0 (ie. HVM without stubdomain) or just the console support?  If it's the latter, the patch name should be something like ```libxl-disable-qemu-console-helpers-in-dom0.patch```.

Reply:
The patch is only disabling QEMU from providing the extra consoles.  Non-stubdom will still launch svirt-interpose/qemu.

The patch name comes from Qubes.  I would not change it, but I can if you definitely want that.